deepchem/data/data_loader.py

```diff
@@ -22,12 +22,12 @@
     from deepchem.feat.dft_data import DFTEntry
     import yaml
     from yaml.loader import SafeLoader
```

Comment:
Instead of a global except move these imports down to the functions they are being used so we avoid the top level import